code/modules/admin/admin_verbs.dm

```diff
@@ -243,7 +243,7 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 
 /client/proc/add_admin_verbs()
 	if(holder)
-		control_freak = CONTROL_FREAK_SKIN | CONTROL_FREAK_MACROS
+		//control_freak = CONTROL_FREAK_SKIN | CONTROL_FREAK_MACROS // Sunset - Adds macro support under client_defines.dm
```

Comment:
Should probably keep `CONTROL_FREAK_SKIN`

Reply:
Went ahead and changed it so that only macros is disabled here